src/hub_predtimechart/hub_config.py

```diff
@@ -112,11 +118,73 @@ def model_output_file_for_ref_date(self, model_id: str, reference_date: str) ->
         return None
 
 
-def _validate_predtimechart_config(ptc_config: dict):
+def _validate_hub_ptc_compatibility(ptc_config: dict, tasks: dict, model_metadata_schema: dict):
     """
-    Validates `ptc_config` against the schema in schema.py.
+    Validates a hub's predtimechart compatibility as identified in README.MD > Assumptions/limitations .
 
     :param ptc_config: dict loaded from a 'predtimechart-config.yml' file
+    :param tasks: dict loaded from a hub's 'tasks.json' file
+    :param model_metadata_schema: dict loaded from a hub's 'model-metadata-schema.json' file
+    :raises ValidationError: if `tasks` is incompatible with predtimechart
+    """
+    # get the round and model_task
+    the_round = tasks['rounds'][ptc_config['rounds_idx']]
+    the_model_task = the_round['model_tasks'][ptc_config['model_tasks_idx']]
+
+    # validate: only `model_tasks` groups with `quantile` output_types will be considered
+    output_type = the_model_task['output_type']
+    if 'quantile' not in output_type:
+        raise ValidationError(f"no quantile output_type found. found types: {list(output_type.keys())}")
+
+    # validate: required quantile levels are present
+    quantile_levels = output_type['quantile']['output_type_id']['required']
+    req_quantile_levels = {0.025, 0.25, 0.5, 0.75, 0.975}
+    if not req_quantile_levels <= set(quantile_levels):  # subset
+        raise ValidationError(f"some quantile output_type_ids are missing. required={req_quantile_levels}, "
+                              f"found={set(quantile_levels)}")
+
+    # validate: model metadata must contain a boolean `designated_model` field
+    if 'designated_model' not in model_metadata_schema['required']:
+        raise ValidationError(f"'designated_model' not found in model metadata schema's 'required' section")
+
+    # validate: in the specified `model_tasks` object within the specified `rounds` object, all objects in the
+    # `target_metadata` list must have the same single key in the `target_keys` object
+    target_keys_keys = set()
+    for target_metadata_obj in the_model_task['target_metadata']:
+        target_keys_keys.update(target_metadata_obj['target_keys'].keys())
+    if len(target_keys_keys) != 1:
+        raise ValidationError(f"more than one unique `target_metadata` key found: {target_keys_keys}")
+
+
+def _validate_predtimechart_config(ptc_config: dict, tasks: dict):
```

Comment:
Same note here about returning a holistic list of errors so people don't have to solve them one by one.